### ISSUE TITLE
atty is unmaintaned and creates an advisory warning

### ISSUE THREAD

**Issue by kgraefe** [borntyping/rust-simple_logger#74]
see https://rustsec.org/advisories/RUSTSEC-2021-0145.html

several projects e.g. [clap](https://github.com/clap-rs/clap/pull/4249) and [env_logger](https://github.com/rust-cli/env_logger/commit/066c2192c187f047ae178e786ff36e6cabed21f7) switched to `is_terminal`

**Comment by V0ldek** [comment]
Correct, running `cargo update -p colored` to update the dep fixes the advisory.

However, I feel like this crate's dep on `colored` should be bumped to minimum of `2.0.4` to enforce this update on upstreams.
